phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/ThemeManager.java

```diff
@@ -67,4 +68,22 @@ public void setDefaultCategoryIconsCss(String defaultCategoryIconsCss) {
     public String getDefaultCategoryIconsCss() {
         return defaultCategoryIconsCss;
     }
+
+    /**
+     * sets the CSS file containing the default engine icons from the repository
+     *
+     * @param defaultEngineIconsCss default engine icons from the repository
+     */
+    public void setDefaultEngineIconsCss(String defaultEngineIconsCss) {
```

Comment:
Shouldn't we use an `URI` object in this case? 
At least this should be added to the javadoc.